### ISSUE TITLE
first stage training

### ISSUE THREAD

**Issue by lavenderrz** [RenYangSCU/ISPDiffuser#2]
Could you clarify how the first stage of the model is trained?

from the paper:
In the first stage, we use paired RAW-sRGB images to train the encoder E(·) and decoder D(·), while freezing the parameters of the diffusion model and HCCM. The encoder and decoder are optimized with the content loss Lstage1 as:
Lstage1 = l2(I − D(E(I)))
where I denotes the input RAW, sRGB, and grayscale images.
The formula 9 in the paper states input I denotes input RAW, sRGB, and grayscale images and output is also I? Do you mean input is RAW and grayscale and output is sRGB?

Also, in the first stage, does the input pass through randomly initialized diffusion model and HCCM

**Comment by RenYangSCU** [comment]
In the first stage, the encoder-decoder is trained to reconstruct the RAW image, sRGB image and grayscale image respectively when using itself as input. I recommend to only train the encoder-decoder in the first stage, and it may be better to train HCCM at the same time.

**Comment by Roger-011025** [comment]
Hi, Thanks for your sharing Author! Do you mean that in the first stage, the training dataset has different kind of data like RAW images(1*H*W), and sRGB images(3*H*W) and graysacle image(1*H*W)? Or you use them together as the input? input = (x, y, y_gray)
